recipes/biorbd-casadi/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+mkdir build
+cd build
+
+cmake .. \
+      -GNinja \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_PREFIX_PATH=$PREFIX \
+      -DMATH_LIBRARY_BACKEND="Casadi" \
+      -DBUILD_EXAMPLE=OFF \
+      -DMODULE_ACTUATORS=ON \
+      -DMODULE_MUSCLES=ON \
+      -DBINDER_PYTHON3=ON \
+        -DNUMPY_INCLUDES=$PREFIX/numpy/core/include \
```

Comment:
You mention numpy includes but don't include it in the dependencies? If you're using the C API take a look at https://conda-forge.org/docs/maintainer/knowledge_base.html#linking-numpy

Reply:
The problem I was referring before ( version that conflicts between CasADi host and the compilation) for some reason were caused by the explicit inclusion of python and numpy.. I'll try do add it only in run to see if it works, but definitely can't for host

Reply:
Ah yes, that's a delightful quirk of staged recipes. Try adding `numpy =1.14` to run and then remove then `=1.14` from the feedstock after this is merged.